File: Cpp/Missing_Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int acSum = 0;
        int exSum = n * (n + 1) / 2;
        for (int val : nums)
            acSum += val;
        return exSum - acSum;
    }
};

File: Rust/Missing_Number.rs
impl Solution {
    pub fn missing_number(nums: Vec<i32>) -> i32 {
        let n = nums.len() as i32;
        let exSum = n * (n + 1) / 2;
        let acSum: i32 = nums.iter().sum();
        return exSum - acSum;
    }
}
